html5/runtime/callback-manager.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * For general callback management of a certain Weex instance.
+ * Because function can not passed into native, so we create callback
+ * callback id for each function and pass the callback id into native
+ * in fact. And when a callback called from native, we can find the real
+ * callback through the callback id we have passed before.
+ */
+export default class CallbackManager {
```

Comment:
You are using ES6 `class` here, although the compiler support it, we haven't use `class` yet. Should the other types make the corresponding change.